Sem. 04/Pract.04/week4_pract_Task12.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+int secondLength(unsigned int second){
```

Comment:
This is a bad name for a function. In general, this function gives the digit count of a number. So countDigits() or length() would be better options